cadence/contracts/DapperUtilityCoin.cdc

```diff
@@ -1,31 +1,21 @@
-import FungibleToken from "./FungibleToken.cdc"
-
-pub contract DapperUtilityCoin: FungibleToken {
-
+import FungibleToken from "FungibleToken"
+access(all) contract DapperUtilityCoin {
     // Total supply of DapperUtilityCoins in existence
-    pub var totalSupply: UFix64
-
+    access(all) var totalSupply: UFix64
     // Event that is emitted when the contract is created
-    pub event TokensInitialized(initialSupply: UFix64)
-
+    access(all) event TokensInitialized(initialSupply: UFix64)
```

Comment:
This can be removed

Reply:
Sorry yeah, the things relying on dapperutilitycoin in this repo were changed prior to the newest token standard updates (This may be true for a few other contracts too). None of the catalog actually relies on dapperutilitycoin aside from the TX generation which i'm not sure if we need to continue to support or not.

The main ones to review are the testing, nftcatalog, retrieval, + nftadmin contracts. The one that the flow.json deploys to previewnet, and the cadence testing goes through